### DISCUSSION TITLE
🇫🇷 List of resources in French language

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/courses#22] [Ideas]
English is not the only language in the world. Many people do not speak English. Many creators do not make content in English. Therefore, we want to create separate lists of resources that can help people who want to build ML knowledge and do not speak that language. French is the next language we want to support.

I am looking for people who would like to help me. I don't know French 😞 I need your help to make it happen.

**Comment by kais-bedioui** [comment]
Hey Piotr,

First of all, Really love and appreciate the work you do for the community.
I look forward to what's upcoming (Maybe performance analysis of Wembanyama
with the Spurs ;)) )

I can help you with setting up French Documentation. However, knowing
french community and education system, English language is the one mostly
used. They study their masters and PhD degrees in English (in ML-related
fields), and a lot of companies are migrating towards English.

But nonetheless, it's a nice initiative to try to include a lot of people
in this. I'll be glad to offer help ;)

Cheers,

Kais

**Reply by SkalskiP** [reply]
Hi @kais-bedioui 👋🏻!

My goal is to make it open to people who are not PhDs, maybe not even undergraduates, but want to teach you something. Funny enough, I majored in civil engineering and only later became interested in Computer Vision.

In addition, I'm not entirely sure, but it seems to me that a large part of Africa speaks French. Do you know if the education system works similarly there?

**Comment by kais-bedioui** [comment]
Hey Piotr,

A big chunk of Africa is indeed francophonic, notably a lot of countries in
North Africa ( Tunisia, Marocco, Algeria) and parts of west Africa & the
Sahara ( No offense but I hope your Geography is good xD )

In Tunisia, my country, I know for sure that almost 99% of universities
teach in French. However English language is becoming more popular, and ML
& CV key terms are well known in their english (original) names than French.

Nonetheless, I'll be glad to help out. I am sure there are open sourced
videos or slides from French/African universites out there on the internet.

How would you like to organize this?

**Reply by SkalskiP** [reply]
> No offense, but I hope your Geography is good

I am rather average! But I know where the countries are haha.

We already have [French README](https://github.com/SkalskiP/courses/blob/master/README.fr.md) thanks to @JimZer 🙏🏻 Now we need to feel that page with links to resources in the French language.

If you have anything worth adding, drop the link here or create a PR.

**Comment by kais-bedioui** [comment]
Alrighty cool

Thanks for hosting such an initiative 🙏🏼

**Comment by kais-bedioui** [comment]
Hey Piotr,

Here's some resources 100% in French & 100% Free with *Entry & Intermediate
levels:*
*1 - *https://openclassrooms.com/fr/search?page=1&query=Machine+Learning
2 -
https://openclassrooms.com/fr/courses/4470531-classez-et-segmentez-des-donnees-visuelles

Course taker should only sign up with this website 'openclassrooms.com'

Let me know if you need extra info ;)

Kais

**Reply by SkalskiP** [reply]
Awesome start! 🔥 Have you had an opportunity to do that course in the past?

**Comment by kais-bedioui** [comment]
I am afraid not, I have personally always relied on English sources.

**Reply by SkalskiP** [reply]
In that case let’s pass. I think we should follow with the same principle as with English resources. We only promote courses that we tested.
